src/app/components/apps-view/apps-view.component.ts

```diff
@@ -79,9 +79,7 @@ export class AppsViewComponent implements OnInit {
       { colId: 'resource', colName: 'Resource', colFormatter: CommonUtil.resourceColumnFormatter },
       { colId: 'queueName', colName: 'Queue Name' },
```

Comment:
Queue name also does not make sense to show under the allocation for an application.
An application cannot run in multiple queues.

Reply:
+1. we can remove the queue name as well

Reply:
@wilfred-s @yangwwei